File: sandbox/fixtures/python-hello/test_hello.py
"""Tests for the hello module."""

import pytest
from hello import greet, add, subtract


def test_greet():
    assert greet("World") == "Hello, World!"


def test_add():
    assert add(2, 3) == 5


def test_subtract():
    assert subtract(5, 3) == 2

File: sandbox/fixtures/ts-hello/test_hello.ts
/** Tests for the hello TypeScript module. */

import { greet, add, subtract } from './hello';

test('greet returns greeting string', () => {
  expect(greet('World')).toBe('Hello, World!');
});

test('add adds two numbers', () => {
  expect(add(2, 3)).toBe(5);
});

test('subtract subtracts two numbers', () => {
  expect(subtract(5, 3)).toBe(2);
});
